build-scripts/rcm-guest/puddle.sh

```diff
@@ -0,0 +1,10 @@
+set -e
+
+CONF_FILE=$(mktemp)
+curl -o "${CONF_FILE}" "${1}"
+
+shift
+
+puddle "${CONF_FILE}" $@
```

Comment:
I would encourage another filename for the wrapper script (e.g. "call_puddle.sh") so that a passing glance does not confuse 'puddle.sh' with 'puddle' .